### ISSUE TITLE
Show recent office hrs video in main landing page or website for better discoverability

### ISSUE THREAD

**Issue by pranay01** [SigNoz/signoz.io#476]

**Comment by Calm-Rock** [comment]
@pranay01 is this still something we would want to have ?

**Comment by pranay01** [comment]
Yes,  especially opentelemetry webinars which we do. but this is more of a website issue than docs. I am adding `growth ideas` label for this. You can ignore this, but let's keep the issue open for future
